nrfx/hal/nrf_gpio.h

```diff
@@ -738,7 +746,7 @@ NRF_STATIC_INLINE uint32_t nrf_gpio_port_impedance_get(NRF_GPIO_Type const * p_r
  * @param p_reg Pointer to the structure of registers of the peripheral.
  * @param mask  Mask of retention domains to be enabled, created using @ref nrf_gpio_retain_mask_t.
  */
-NRF_STATIC_INLINE void nrf_gpio_port_retain_set(NRF_GPIO_Type * p_reg, uint32_t mask);
+NRF_STATIC_INLINE void nrf_gpio_port_retain_write(NRF_GPIO_Type * p_reg, uint32_t mask);
```

Comment:
This should stay as _set

Reply:
we can't, as there's `RETAIN` and `RETAINSET`. There appears to be a precedent with OUT/OUTSET.

Reply:
>There appears to be a precedent with OUT/OUTSET.

That API is old and needs to be aligned in the future. Current approach is to use `set`, `enable` and `disable`.